connector-drawer/src/main/java/com/redhat/cloud/notifications/connector/drawer/DrawerPayloadBuilder.java

```diff
@@ -48,7 +47,7 @@ public void process(Exchange exchange) {
         outgoingCloudEvent.put("specversion", CE_SPEC_VERSION);
         outgoingCloudEvent.put("datacontenttype", "application/json");
         outgoingCloudEvent.put("source", "urn:redhat:source:notifications:drawer");
-        outgoingCloudEvent.put("id", entry.getId());
+        outgoingCloudEvent.put("id", entryPayloadModel.getId());
```

Comment:
This change would make it obvious what the data we're sending actually is:
```suggestion
        outgoingCloudEvent.put("id", entryPayloadModel.getEventId());
```